crates/configuration/src/parachain.rs

```diff
@@ -23,14 +24,24 @@ pub struct ParachainConfig {
     id: u32,
 
     /// Chain to use (use None if you are running adder-collator or undying-collator).
-    chain: Option<Chain>,
+    chain: Chain,
 
     /// Registration strategy for the parachain.
     registration_strategy: Option<RegistrationStrategy>,
 
     /// Parachain balance.
     initial_balance: u128,
 
+    default_command: Option<Command>,
+
+    default_image: Option<Image>,
+
+    default_resources: Option<Resources>,
+
+    default_db_snapshot: Option<AssetLocation>,
+
+    default_args: Vec<Arg>,
+
```

Comment:
I didn't put comment here voluntary, the refacto of the doc will come in the next PR fyi